guides/kubernetes/kubernetes.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes
+
+Kubernates(also written as K8s) is an open source system for automatic , scaling and managemnt of [containerized applications](https://searchitoperations.techtarget.com/definition/application-containerization-app-containerization). It is the most popular tool among all currently present which is used mostly by all the major giants of the industry including Google who actually founded it, Amazon and Microsoft etc.
+
+The Github repository of kubernates project can be visited from [here](https://github.com/kubernetes/kubernetes)
+
+Before knowing it further you must be aware of what actually a docker is,what are containers and containerized applications. 
+
+## Why kubernetes ?
+
+As i mentioned that kubernates is a product of google which was released in July 2015 after a decade and half year effort of google who was earlier facing with its own internal Borg system which had certain drawbacks. So they thought of creating something new which can be standarized which can be used by all and the result is kubernetes.
```

Comment:
1) Please refrain from writing these guides from the first person. So, remove "As I mentioned".
2) Can you please change the last sentence - "So they thought of creating something new which can be standardized which can be used by all and the result is kubernetes" to "So, they created an open-source container-orchestration system called, Kubernetes to be useful for all."